calculadora_do_cidadao/adapters/cpi.py

```diff
@@ -0,0 +1,81 @@
+from datetime import date
+from decimal import Decimal
+from typing import NamedTuple
+from urllib.parse import urlencode
+
+from rows.fields import DecimalField, DateField
+
+from calculadora_do_cidadao.adapters import Adapter
+from calculadora_do_cidadao.months import MONTHS
+from calculadora_do_cidadao.typing import MaybeIndexesGenerator
+
+
+today_date = str(date.today())
```

Comment:
Is this a constant? Any special reason for not having it in uppercase?

Reply:
No reason at all... And yes, it's a constant!